frontend/src/pages/org/archived-item-detail/archived-item-detail.ts

```diff
@@ -123,17 +158,35 @@ export class CrawlDetail extends LiteElement {
       void this.fetchCrawl();
       void this.fetchCrawlLogs();
       void this.fetchSeeds();
+
+      this.fetchTabData();
+    } else {
+      if (changedProperties.has("activeTab") && this.activeTab) {
```

Comment:
is `this.activeTab` ever going to be falsey?

Reply:
Not sure, but maybe revisit later for this one?

Reply:
Want to err on side of caution for now.